protocols/protocol/cloudstate/crdt.proto

```diff
@@ -92,8 +118,20 @@ message CrdtStreamIn {
     }
 }
 
-// Message for the Crdt handle stream out.
+//*CrdtStreamOut* TODO
+//
+//---
+//
+//`Specification`:
+//
+//  - A `CrdtStreamOut` is considered to be valid iff:
+//    - `message` MUST be one of the following:
+//      - `reply` TODO
+//      - `streamed_message` TODO
+//      - `stream_cancelled_response` TODO
+//      - `failure` TODO
 message CrdtStreamOut {
+
```

Comment:
I think it's not entirely clear what should be replied for a non command incoming message, solely by reading the spec.

CrdtReply is commented with "A reply to an incoming command." Any of the CrdtStreamIn not beeing the Command message are not command messages, right?
In the Crdt service it's stated that: "The user function must respond with one reply per command in." so this applies to Command messages only, yes?